tests/unittests/test_extension.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+import sys
+
+from fontTools.misc.py23 import tounicode
+
+from psautohint import _psautohint
+
+
+INFO = b"FontName Foo"
+NAME = b"Foo"
+GLYPH = b"""% square
+0 500 rb
+60 500 ry
+sc
+560 500 mt
+560 0 dt
+60 0 dt
+60 500 dt
+cp
+ed
+"""
+
+
+def test_autohint_good_args():
+    _psautohint.autohint(INFO, GLYPH)
+
+
+def test_autohintmm_good_args():
+    _psautohint.autohintmm(INFO, (GLYPH, GLYPH), (NAME, NAME))
+
+
+@pytest.mark.parametrize("args", [
+  [],                          # no arguments
```

Comment:
this and many other lines below the `@pytest.mark` blocks in this file are indented with only 2 spaces instead of 4

Reply:
That is because some lines had too long comments, I’m rewording them now to be shorter.